modules/network-observability-multitenancy.adoc

```diff
@@ -3,22 +3,43 @@
 // network_observability/installing-operators.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="network-observability-multi-tenancy{context}"]
+[id="network-observability-multi-tenancy_{context}"]
 = Enabling multi-tenancy in Network Observability
-Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project administrators. Project administrators who have limited access to some namespaces can access flows for only those namespaces. 
+
+For Developers, multi-tenancy is available for both Loki and Prometheus but requires different access rights.
 
 .Prerequisite
-* You have installed at least link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]
+* If you are using Loki, you have installed at least link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7]
 * You must be logged in as a project administrator
 
 .Procedure
 
-. Authorize reading permission to `user1` by running the following command:
+* For per-tenant access, you must have the `netobserv-reader` cluster role and the `netobserv-metrics-reader` namespace role. In this scenario, it is preferable to use the developer perspective. The administrator perspective only works with Loki. Run the following commands for this level of access:
```

Comment:
```suggestion
* For per-tenant access, use the developer perspective. The administrator perspective only works with Loki. Run the following commands to get the per-tenant level of access:
```
- This can be updated to say something like -- 
"for this level of access" -- "to get the per-tenant level of access" 
- This  info "For per-tenant access, you must have the `netobserv-reader` cluster role and the `netobserv-metrics-reader` namespace role" can be removed from here and be used as instructions for the later commands. For example - "Add the netobserv-reader cluster role to.....by running the following..."